go-controller/pkg/ovn/egressip.go

```diff
@@ -1958,6 +2072,34 @@ type egressIPController struct {
 	egressIPNodeHealthCheckPort int
 }
 
+// addStandByEgressIPAssignment does the same setup that is done by addPodEgressIPAssignments but for
+// the standby egressIP. This must always be called with a lock on podAssignmentState mutex
+// This is special case function called only from deleteEgressIPAssignments, don't use this for normal setup
+func (oc *Controller) addStandByEgressIPAssignment(podKey string, podStatus *podAssignmentState) error {
+	podNamespace, podName := getPodNamespaceAndNameFromKey(podKey)
+	pod, err := oc.watchFactory.GetPod(podNamespace, podName)
+	if err != nil {
+		return err
+	}
+	// TODO: Loop through multiple standby's example if the selected EIP doesn't have any statuses
+	eipToAssign := podStatus.standbyEgressIPNames.UnsortedList()[0]
+	eip, err := oc.watchFactory.GetEgressIP(eipToAssign)
+	if err != nil {
+		return err
```

Comment:
on second thoughts I am thinking .... should we ever retry if there is error from `addStandByEgressIPAssignment`? At this point I feel like it should be  best effort and at some later point things would sync, but retrying deleting because addition failed seems wrong...